pubspec.lock

```diff
@@ -8,6 +8,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.0.2"
+  archive:
```

Comment:
Changes to this file should not be committed. Please remove them.

Reply:
These are automatically generated files imo we should add this to .gitignore.
Everytime packages get is done, these files are generated.